IDEAS/Buildings/Components/Examples/horizontalFins.mo

```diff
@@ -0,0 +1,63 @@
+within IDEAS.Buildings.Components.Examples;
```

Comment:
@wildw0lf can you remove this example and `HFinsAndOverhang`, and instead add `HorizontalFins` to `Examples.BuildingShadeExample`? This way we test all shading models in a single model, which simplifies inter comparison of the results.

For future reference, all classes should start with a capital letter :)